packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1268,73 +1367,6 @@ contract GenArt721CoreV3 is
         return projectIdToFinancials[_projectId].artistAddress;
     }
 
-    /**
-     * @notice View function returning Artist's secondary market royalty
-     * percentage for project `_projectId`.
-     * This does not include Art Blocks portion of secondary market royalties.
-     * @param _projectId Project ID to be queried.
-     * @return uint256 Artist's secondary market royalty percentage.
-     */
-    function projectIdToSecondaryMarketRoyaltyPercentage(
```

Comment:
removing is to reduce contract bytecode for duplicate accessor - requires subgraph follow-on in `mapping-v3-core.ts:handleProjectSecondaryMarketRoyaltyPercentageUpdated` function to add another try in the case where trying to call the removed function fails.

Reply:
I ended up re-adding this because we now have sufficient bytecode margin, and re-adding removes some undesirable interface complexity in subgraph